.github/workflows/dan_layer_integration_tests.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           command: build
           args: --release --bin tari_validator_node
       - name: npm ci
-        run: cd integration_tests && npm ci && cd node_modules/wallet-grpc-client && npm ci
+        run: cd integration_tests && npm ci && cd node_modules/wallet-grpc-client && npm install
```

Comment:
Doing this means npm might resolve new packages instead of what’s in package-lock

Reply:
Currently the package-lock is gitignored. So the best solution is likely to commit  it.

Reply:
Yeah I thought so but I think Mike removed them a while back? 🤷‍♂️

Reply:
lol it gets deleted every now and again. It should be included

Reply:
Added!